package.json

```diff
@@ -43,6 +43,7 @@
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
     "test-no-cover": "repo-tools test run --cmd mocha -- build/test/*.js --no-timeouts",
     "test": "repo-tools test run --cmd npm -- run cover",
+    "test-docker": "docker build . -t test && docker run -it test",
```

Comment:
hmm, looks like this should be in a separate pr, shouldn't it?

Reply:
Fair point :)